### PR TITLE
feat: sort by last updated

### LINKED ISSUE #139: Sorting by "recently added"
Hi 👋🏻 

I would be nice for nuxt lovers to be able to know when a new module is added, easier to keep up with new modules and helpful for learning curve.

Thanks.

### PR DESCRIPTION
**Pull request by antfu**
resolves #139

image

### CODE REVIEW

File: website/server/api/modules.ts [hunk 1]
```diff
@@ -21,9 +22,11 @@ async function fetchModuleStats (module) {
     ])
     module.downloads = npm.downloads.lastMonth
     module.stars = github.stars
+    module.updatedAt = +new Date(npm.updatedAt || undefined)
```
Comment (pi0) [inline website/server/api/modules.ts:25]: I would suggest using `publishedAt` field to reflect `latest` tag

File: website/composables/constants.ts [hunk 2]
```diff
@@ -28,7 +28,8 @@ export const VERSIONS = [
 
 export const ORDER_BY_FIELDS = {
   downloads: 'Downloads',
-  stars: 'Stars'
+  stars: 'Stars',
+  updatedAt: 'Updated'
```
Comment (pi0) [inline website/composables/constants.ts:32]: Maybe also allow sorting by creation time?

File: website/server/api/modules.ts (hunk 1 above)
Comment (antfu) [reply website/server/api/modules.ts:25]: Sorry what do you mean?

Comment (pi0) [reply website/server/api/modules.ts:25]:
```suggestion
    module.publishedAt= +new Date(npm.publishedAt || undefined)
```

updatedAt is from npm registry DB. Basically any change to registry changes that field which is less accurate if we want to sort by last version time

Comment (antfu) [reply website/server/api/modules.ts]: Oh I misread - thanks!

**Review by pi0 — APPROVED** [review]

**Pull request merged**